pkg/deploy/request.go

```diff
@@ -111,6 +114,10 @@ func getFiles(dirFileNames []string) ([]string, error) {
 	return files, nil
 }
 
+func isHTTPUrl(fileName string) bool {
+	return strings.Index(fileName, "http://") == 0 || strings.Index(fileName, "https://") == 0
```

Comment:
probably overkill, but shall we validate the url using regexp against http://www.faqs.org/rfcs/rfc3987.html ?

Reply:
I switched this to using the url.Parse library and then eliminiating relative urls which would give false positive